sa/sysvars.go

```diff
@@ -0,0 +1,219 @@
+		"histogram_size":                       {},
+		"idle_readonly_transaction_timeout":    {},
+		"idle_transaction_timeout":             {},
+		"idle_write_transaction_timeout":       {},
+		"in_predicate_conversion_threshold":    {},
+		"insert_id":                            {},
+		"interactive_timeout":                  {},
+		"join_buffer_size":                     {},
+		"join_buffer_space_limit":              {},
+		"last_insert_id":                       {},
+		"lock_wait_timeout":                    {},
+		"log_slow_min_examined_row_limit":      {},
+		"log_slow_query_time":                  {},
+		"log_slow_rate_limit":                  {},
+		"long_query_time":                      {},
+		"max_allowed_packet":                   {},
+		"max_delayed_threads":                  {},
+		"max_digest_length":                    {},
+		"max_error_count":                      {},
+		"max_heap_table_size":                  {},
+		"max_join_size":                        {},
+		"max_length_for_sort_data":             {},
+		"max_recursive_iterations":             {},
+		"max_rowid_filter_size":                {},
+		"max_seeks_for_key":                    {},
+		"max_session_mem_used":                 {},
+		"max_sort_length":                      {},
+		"max_sp_recursion_depth":               {},
+		"max_statement_time":                   {},
+		"max_user_connections":                 {},
+		"min_examined_row_limit":               {},
+		"mrr_buffer_size":                      {},
+		"net_buffer_length":                    {},
+		"net_read_timeout":                     {},
+		"net_retry_count":                      {},
+		"net_write_timeout":                    {},
+		"optimizer_extra_pruning_depth":        {},
+		"optimizer_max_sel_arg_weight":         {},
+		"optimizer_prune_level":                {},
+		"optimizer_search_depth":               {},
+		"optimizer_selectivity_sampling_limit": {},
+		"optimizer_trace_max_mem_size":         {},
+		"preload_buffer_size":                  {},
+		"profiling_history_size":               {},
+		"progress_report_time":                 {},
+		"pseudo_slave_mode":                    {},
+		"pseudo_thread_id":                     {},
+		"query_alloc_block_size":               {},
+		"query_prealloc_size":                  {},
+		"rand_seed1":                           {},
+		"range_alloc_block_size":               {},
+		"read_rnd_buffer_size":                 {},
+		"rowid_merge_buff_size":                {},
+		"sql_select_limit":                     {},
+		"tmp_disk_table_size":                  {},
+		"transaction_alloc_block_size":         {},
+		"transaction_prealloc_size":            {},
+		"wait_timeout":                         {},
+		"warning_count":                        {},
+	}
+
+	if _, found := mariaDBNumericTypes[name]; found {
+		if checkNumericRE.FindString(value) != value {
+			return fmt.Errorf("%s requires numeric value, but %s is not formatted like a number", name, value)
```

Comment:
While I'm flattered that you liked my suggestion for the error message here, it's now the only one that doesn't follow the format set by the others :D In general, consistency is a priority, so either this one should still match the others, or they should all take this form.

Reply:
Fixing it now